src/Monocle/Backend/Index.hs

```diff
@@ -77,6 +77,19 @@ instance ToJSON DateIndexMapping where
       , "format" .= ("date_time_no_millis" :: Text)
       ]
 
+-- https://www.elastic.co/guide/en/elasticsearch/reference/7.17/enabled.html#enabled
+data NonIndexedMapping = NonIndexedMapping deriving (Eq, Show)
+
+instance ToJSON NonIndexedMapping where
+  toJSON NonIndexedMapping =
+    object
+      [ "type" .= ("object" :: Text)
+      , "enabled" .= (False :: Bool)
+      ]
+
+mergedCommitField :: [Pair]
+mergedCommitField = ["merged_commit_sha" .= NonIndexedMapping]
```

Comment:
Perhaps this should be indexed, so that later we can do filter based on this field?
@leonid-deriv would that works for your use-case?

Reply:
Well, I've updated the PR to use the standard keyword mapping for that field.